packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -13,12 +14,132 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
       backgroundColor: t.colors.transparentBackground,
       color: t.colors.brandForeground1,
       borderColor: t.colors.brandForeground1,
-      iconColor: t.colors.brandedBorder,
+      iconColor: t.colors.brandForeground1,
     },
     tint: {
-      backgroundColor: t.colors.brandedBackground,
-      color: t.colors.brandForeground2,
-      borderColor: t.colors.brandForeground2,
-      iconColor: t.colors.brandStroke2,
+      backgroundColor: globalTokens.color.brand.tint60,
+      color: t.colors.brandForeground1,
+      borderColor: '#B4D6FA',
```

Comment:
This color is based off of Outlook's color while our base brand token is based off of Word. Hard coding it like this also means it won't change color for other Office apps.

I would suggest t.colors.brandStroke2. It's not a perfect match but if you work off the alias token, it'll switch out the brand color based on the host app

Reply:
Should I change createAliasesFromPalette.ts for that? To add those colors there

Reply:
All the brand tokens are overwritten in createBrandedThemeWithAlias